PKHeX.WinForms/PKHeX.WinForms.csproj

```diff
@@ -1,1065 +1,60 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
   <PropertyGroup>
```

Comment:
You can add a <Version></Version> tag to set the assembly version, which probably fixes whatever issues you were having with update checking.

I just updated the Azure pipeline to set the version on build.